jhotdraw-core/src/main/java/org/jhotdraw/draw/handle/FontSizeHandle.java

```diff
@@ -133,18 +133,14 @@ public void keyPressed(KeyEvent evt) {
         oldSize = newSize = textOwner.getFontSize();
         switch (evt.getKeyCode()) {
             case KeyEvent.VK_UP:
-                if (newSize > 1) {
-                    newSize -= 1f;
-                }
+                if (newSize > 1) newSize -= 1f;
```

Comment:
brackets - maybe you should adapt your formatting settings. Unfortunately I have no checkstyle plugin attached to the project.